src/Hazelcast.Net/Serialization/Compact/Schema.cs

```diff
@@ -131,10 +127,16 @@ private void Initialize(string typeName, IEnumerable<SchemaField> typeFields)
             {
```

Comment:
See the comment above this line: `// build the fields map, which is sorted (see above)`. Is it correct? I don't think typeFields is sorted here.